app/templates/application.gts

```diff
@@ -1,6 +1,7 @@
 import Route from 'ember-route-template';
 import { pageTitle } from 'ember-page-title';
-import { WelcomePage } from 'ember-welcome-page';
+//import { WelcomePage } from 'ember-welcome-page';
```

Comment:
afaict, this should still work, just poking around here because:
```
Error: Build failed with 1 error:
node_modules/.pnpm/typescript-memoize@1.1.1/node_modules/typescript-memoize/dist/memoize-decorator.js:132:51: ERROR: [plugin: embroider-esbuild-resolver] /Users/psego/Development/tmp/vite-app/node_modules/.pnpm/ember-welcome-page@7.0.2/node_modules/ember-welcome-page/dist/components/welcome-page.js: this.config.activePackageRules is not iterable
    at failureErrorWithLog (/Users/psego/Development/tmp/vite-app/node_modules/.pnpm/esbuild@0.21.5/node_modules/esbuild/lib/main.js:1472:15)
    at /Users/psego/Development/tmp/vite-app/node_modules/.pnpm/esbuild@0.21.5/node_modules/esbuild/lib/main.js:945:25
    at <repo>/node_modules/.pnpm/esbuild@0.21.5/node_modules/esbuild/lib/main.js:1353:9
```